.github/workflows/CD.yml

```
@@ -31,3 +31,39 @@ jobs:
           user: accessnri
           label: main
           token: ${{ secrets.anaconda_token }}
+
+  pypi-build:
+    name: Build package for PyPI
+    if: github.repository == 'payu-org/payu' && github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v3
```

This can now be v4

---

```suggestion
      - uses: actions/setup-python@v4
```
Suggestions are great for this sort of small change.

---

True. Annoyingly, I don't think I can make suggestions on code that hasn't changed (e.g. the `actions/checkout@v3` used in the `conda` job)?

---

> I don't think I can make suggestions on code that hasn't changed

Yeah I think you're right.

---

https://github.com/github/roadmap/issues/347